devices/RgbDiode/samples/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="nanoFramework.CoreLibrary" version="1.15.5" targetFramework="netnano1.0" />
```

Comment:
you need as well the graphics core. Because you are in project reference, it's going to be deployed, if not, it needs to be added (theoretically, it will automatically with the nuget), still better to have all the ones needed.

Reply:
Added all missing.

Reply:
Actually, after adding it the build fails. Reverting.

Reply:
you need to add them in the nfproj, they will appear here.
So add the nuget from Visual Studio and all good.

Reply:
done